packages/treasury/src/stablecoinMachine.js

```diff
@@ -131,6 +168,49 @@ export async function start(zcf) {
     const collateralBrand = zcf.getBrandForIssuer(collateralIssuer);
     assert(!collateralTypes.has(collateralBrand));
 
+    const {
+      publicFacet: rateManagerPublic,
+      manager: rateManager,
+    } = buildParamManager([
```

Comment:
this has got to go into a different file or something. Or we need a much more abbreviated syntax for this.

Reply:
I'm thinking each param should be a separate object, rather than havign them all looked up through essentially a map. 
```
const chargingPeriodParam = gov.natParam(CHARGING_PERIOD_KEY, loanParams.chargingPeriod);
const recordingPeriodParam = gov.natParam(RECORDING_PERIOD_KEY, loanParams.recordingPeriod);
```

and then usage would be `chargingPeriodParam.value()` or some such.